service/ioc-start.sh

```diff
@@ -15,5 +15,14 @@ done
 case "$(decode-reg build_info -q --slot ${devslot})" in
     afc-tim-receive*|afcv4_fofb_ctrl*|pbpm-gw*|bpm-gw*)
         systemctl restart afc-ioc@${devslot}.service
+        ;;&
+    bpm-gw*)
+        VSLOT1=$(/opt/afc-epics-ioc/iocBoot/iocutca/getSlot.sh $devslot)
+        VSLOT2=$(( VSLOT1 + 1 ))
+
+        services="rffe-ioc@${VSLOT1}.service"
+        [ "$VSLOT2" -lt 24 ] && services+=" rffe-ioc@${VSLOT2}.service"
+
+        systemctl restart $services
```

Comment:
As we discussed, I think this should be `start` instead. The RFFE hardware isn't resilient to closing and opening connections repeatedly.